cli/cmdlineparser.cpp

```diff
@@ -973,6 +976,9 @@ void CmdLineParser::printHelp()
               "Options:\n"
               "    --addon=<addon>\n"
               "                         Execute addon. i.e. cert.\n"
+              "    --python=<python interpreter>\n"
```

Comment:
I can understand why you put this here, it's logical. but we put the options in alphabetical order. I would suggest you put the options in alphabetical order but for `--addon` we can write something like `see also --python`.

Reply:
or maybe rename this option to `--addon-python`

Reply:
I'll rename to --addon-python